src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Internal/JQueryKeyValuePairNormalizerToMvc.cs

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.AspNetCore.Mvc.Core;
+using Microsoft.Extensions.Primitives;
+
+namespace Microsoft.AspNetCore.Mvc.ModelBinding.Internal
```

Comment:
When using `internal` for realz, put this in a real namespace. Suggest `Microsoft.AspNetCore.Mvc.ModelBinding`.